misc/admin/lib/github.js

```diff
@@ -157,10 +157,11 @@ function _createRelease(user, password, tagName, title, body, prerelease, commit
             })),
             method: "POST",
             headers: {
-                "User-Agent": "hashgraph/hethers"
+                "User-Agent": "ethers-io"
```

Comment:
Is this change commited by mistake?

Reply:
No, there seemed to be a problem with this function and this is an experiment to see if it will work correctly this time.